src/js/videojs-dash.js

```diff
@@ -73,11 +73,18 @@ class Html5DashJS {
     // element to bind to.
     this.mediaPlayer_.initialize();
 
-    // Apply any options that are set
-    if (options.dash && options.dash.limitBitrateByPortal) {
```

Comment:
My mistake, I just noticed you're prepending 'set' to each of the key options. No need for a deprecation warning.